charts/dex.yaml

```diff
@@ -52,8 +52,8 @@ config:
     tlsCert: /etc/dex/tls/https/server/tls.crt
     tlsKey: /etc/dex/tls/https/server/tls.key
   frontend:
-    theme: "coreos" #metalk8s-ui
-    # dir: /web/themes/custom/
+    theme: "scality"
+    issuer: "MetalK8s"
```

Comment:
Must have took you quite some time to find out how this works, as it's not documented at all... I had to read the Go sources to understand what it does... Maybe worth a PR upstream to clarify this usage?